.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,6 +19,9 @@ Note: If merging this PR should also close the associated Issue, please also add
 
 **Additionally, please run the sagemaker local tests in at least one revision:**
 - [ ] `sagemaker_local_tests = true`
+  
+### Formatting
+I have run `black -l 100` on my code (formatting tool: https://black.readthedocs.io/en/stable/getting_started.html)
```

Comment:
Is this meant to be a checkbox? If so, it's missing.

Reply:
It prob makes more sense as a checkbox. Added.